### PR TITLE
Fix casing in Dropdown.js

### LINKED ISSUE #351: Non-camelcase prop
See https://github.com/react-materialize/react-materialize/blob/master/src/Dropdown.js#L67

Shouldn't `constrain_width` be `constrainWidth` ?

http://materializecss.com/dropdown.html#options

### PR DESCRIPTION
**Pull request by lloydwatkin**
Fixes #351

Confirmed the camelCased property works

### CODE REVIEW

**Comment by lloydwatkin** [comment]
Have pushed an update.

File: test/DropdownButtonSpec.js [hunk 1]
```diff
@@ -19,7 +20,19 @@ describe('<Dropdown />', () => {
     assert.ok(wrapper.find('.dropdown-content').hasClass('more'));
   });
 
-  it('doesn\'t pass down unwatned props', () => {
+  it('passed `constrainWidth` correctly', () => {
+    const options = { constrainWidth: true };
+    const dropdown = spy();
+    stub(global, '$', () => { return { dropdown }; });
```
Comment (alextrastero) [inline test/DropdownButtonSpec.js]: No need for this has dropdown is already stubbed in the helper, have a look at this example: https://github.com/react-materialize/react-materialize/blob/e34af17a58cedf9a8a22b3c9ab6095537a4ffd2f/test/CarouselSpec.js#L33

**Comment by lloydwatkin** [comment]
Thanks, have updated.

**Comment by alextrastero** [comment]
Perfect! thanks

**Comment by lloydwatkin** [comment]
Great, do you have an idea of when the next release will be? Thanks!

**Comment by alextrastero** [comment]
As soon as this PR is approved https://github.com/react-materialize/react-materialize/pull/360

**Pull request merged**